pkg/frontend/admin_platformworkloadidentityroleset_list_test.go

```diff
@@ -0,0 +1,159 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/admin"
+	"github.com/Azure/ARO-RP/pkg/metrics/noop"
+	testdatabase "github.com/Azure/ARO-RP/test/database"
+)
+
+func TestPlatformWorkloadIdentityRoleSetList(t *testing.T) {
+	ctx := context.Background()
+
+	type test struct {
+		name           string
+		fixture        func(f *testdatabase.Fixture)
+		wantStatusCode int
+		wantResponse   *admin.PlatformWorkloadIdentityRoleSetList
+		wantError      string
+	}
+
+	for _, tt := range []*test{
+		{
+			name:           "empty",
```

Comment:
nit: it is clearer to name test cases with expected prereqs and outcomes. For example, 6mo down the line when this fails, `empty` is a lot less clear than `empty list of WorkloadIdentityRoleSet cosmos documents returns StatusOK` due to some other change in the code elsewhere. I would add the same nit for all other test cases below.